static/postData.ts

```diff
@@ -0,0 +1,49 @@
+function getData(): string {
```

Comment:
I don't really understand what this is returning. It seems to setting up the configuration for what is going to be used to request the data? in which case getDataParams() might be better

Reply:
I think it probably needs a better name, and an example of what it is returning